src/coreclr/jit/emitxarch.cpp

```diff
@@ -6770,8 +6780,13 @@ void emitter::emitIns_R_AR_R(instruction ins,
     emitCurIGsize += sz;
 }
 
-void emitter::emitIns_R_R_C(
-    instruction ins, emitAttr attr, regNumber reg1, regNumber reg2, CORINFO_FIELD_HANDLE fldHnd, int offs)
+void emitter::emitIns_R_R_C(instruction          ins,
+                            emitAttr             attr,
+                            regNumber            reg1,
+                            regNumber            reg2,
+                            CORINFO_FIELD_HANDLE fldHnd,
+                            int                  offs,
+                            bool                 isEmbBroadcast)
```

Comment:
We should probably make this a more general purpose `insOpts` like we have on Arm64. That way we can easily extend it to other important EVEX information.